lib/updateMaterialProperties.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+var Cesium = require('cesium');
+var addExtensionsUsed = require('./addExtensionsUsed');
+var ForEach = require('./ForEach');
+
+var defined = Cesium.defined;
+var WebGLConstants = Cesium.WebGLConstants;
+
+module.exports = updateMaterialProperties;
+
+function isStateEnabled(renderStates, state) {
+    return (renderStates.enable.indexOf(state) > -1);
+}
+
+/**
+ * Move glTF 1.0 technique render states to glTF 2.0 materials properties and EXT_blend extension.
+ *
+ * @param {Object} gltf A javascript object containing a glTF asset.
+ * @returns {Object} The updated glTF asset.
+ *
+ * @private
+ */
+function updateMaterialProperties(gltf) {
+    var blendingForTechnique = {};
+    var materialPropertiesForTechnique = {};
+    var techniquesLegacy = gltf.techniques;
+    if (!defined(techniquesLegacy)) {
+        return gltf;
+    }
+
+    ForEach.technique(gltf, function (techniqueLegacy, techniqueIndex) {
+        var renderStates = techniqueLegacy.states;
+        if (defined(renderStates)) {
+            var materialProperties = materialPropertiesForTechnique[techniqueIndex] = {};
+
+            // If BLEND is enabled, the material should have alpha mode BLEND
+            // and save the blend functions to the EXT_blend extension
+            if (isStateEnabled(renderStates, WebGLConstants.BLEND)
+                    || defined(renderStates.blendEquationSeparate) || defined(renderStates.blendFuncSeparate)) {
+                materialProperties.alphaMode = 'BLEND';
+                if (defined(renderStates.functions)) {
+                    blendingForTechnique[techniqueIndex] = {
+                        blendEquation: renderStates.functions.blendEquationSeparate,
+                        blendFactors: renderStates.functions.blendFuncSeparate
```

Comment:
It looks like gltf 1 has more blend factors than the draft extension.

```
ZERO
ONE
SRC_COLOR
ONE_MINUS_SRC_COLOR
SRC_ALPHA
ONE_MINUS_SRC_ALPHA
DST_ALPHA
ONE_MINUS_DST_ALPHA
DST_COLOR
ONE_MINUS_DST_COLOR
CONSTANT_COLOR
ONE_MINUS_CONSTANT_COLOR
CONSTANT_ALPHA
ONE_MINUS_CONSTANT_ALPHA
SRC_ALPHA_SATURATE
```
vs

```

ZERO               
ONE                
SRC_COLOR          
ONE_MINUS_SRC_COLOR
SRC_ALPHA          
ONE_MINUS_SRC_ALPHA
DST_ALPHA          
ONE_MINUS_DST_ALPHA
DST_COLOR          
ONE_MINUS_DST_COLOR
```

If one of the old ones are encountered maybe just switch to the default value? Or not add the extension?